README.md

```diff
@@ -539,6 +539,104 @@ pathfinder:
           pathType: Prefix
 ```
 
+### Deploying Sombra on Azure
+
+The following example deploys Sombra on Azure. This would require setting up an Application Gateway, which will be used as the ingress controller, on Azure and adding the TLS secret into the cluster.
+
+```yaml
+imageCredentials:
+  registry: docker.transcend.io
+  username: Transcend
+  password: '<TRANSCEND_API_TOKEN>'
+
+transcend_service:
+  type: NodePort
```

Comment:
Also I feel this will work with default `ClusterIP` too

Reply:
as in we can change the type to ClusterIP?

Reply:
yes, basically you remove this `transcend_service` and `customer_service` section and let chart use default one. Please do test it once